suite-native/storage/package.json

```diff
@@ -12,6 +12,8 @@
     },
     "dependencies": {
         "@reduxjs/toolkit": "1.8.3",
+        "expo-random": "^13.0.0",
```

Comment:
You should add all native dependencies also to `suite-native/app/package.json` otherwise there could be problems that their will be missed by autolinking etc. I not sure how it's possible that it works now :D

Reply:
Fixed in de5504964